lib/scheduler.js

```diff
@@ -8,11 +8,11 @@ var { storageManager } = require('./storage-manager.js');
 var { toolbarButton } = require('./toolbar-button.js');
 var { utils } = require('./utils.js');
 
+var destroyTimer;
```

Comment:
This is a bit of a bike-sheddy thing, but will likely help me understand for future reviews. What's the rationale behind using `var` still in ES6-land?